lib/ansible_tower_client/inventory.rb

```diff
@@ -4,6 +4,16 @@ def self.endpoint
       "inventories".freeze
     end
 
+    def inventory_sources
+      api.inventories.find_all_by_url(related['inventory_sources'])
```

Comment:
`api.inventories.find(inventory_sources)` seems misleading.  I think you want to do something similar to the relations on other classes.